src/client.js

```diff
@@ -3,6 +3,7 @@ const { MONDAY_OAUTH_URL } = require("./constants.js");
 const { convertToArrayIfNeeded } = require("./helpers");
 const { initScrollHelperIfNeeded } = require("./helpers/ui-helpers");
 const { initBackgroundTracking } = require("./services/background-tracking-service");
+const { logWarnings } = require("./helpers/monday-api-helpers");
```

Comment:
general comment: need to make this a patch version, i think we should set the version manually